app/views/challenge/prompt_meme/confirm_delete.html.erb

```diff
@@ -0,0 +1,6 @@
+<!--Descriptive page name, messages and instructions-->
+<h2 class="heading"><%= ts("Delete Prompt Meme") %></h2>
+
+<!--main content-->
+<%= render "challenge/shared/challenge_form_confirm_delete" %>
+<!--/content-->
```

Comment:
Could you add a final newline to this file, `app/views/challenge/shared/_challenge_form_confirm_delete.html.erb`, and `app/views/challenge/shared/_challenge_form_delete.html.erb`? (Hound would ordinarily comment asking for that, but it doesn't check views.)